core/src/test/java/com/google/errorprone/bugpatterns/DefaultCharsetTest.java

```diff
@@ -389,8 +389,8 @@ public void printWriter() {
             "        new BufferedWriter(new OutputStreamWriter(System.err, UTF_8)), true);",
             "    PrintWriter pw2 = new PrintWriter(",
             "        new BufferedWriter(new OutputStreamWriter(System.err, UTF_8)));",
-            "    PrintWriter pw3 = new PrintWriter(\"test\", UTF_8.name());",
-            "    PrintWriter pw4 = new PrintWriter(new File(\"test\"), UTF_8.name());",
+            "    PrintWriter pw3 = new PrintWriter(\"test\", UTF_8);",
+            "    PrintWriter pw4 = new PrintWriter(new File(\"test\"), UTF_8);",
```

Comment:
I missed that the change was only updating the refactoring for `Scanner`, not `PrintWriter`

Can you also remove the use of `.name()` on L547 in `handlePrintWriter`?